forecasttools/daily_to_epiweekly.py

```diff
@@ -1,12 +1,24 @@
+"""
+Converts daily resolution dataframes
+to epiweekly dataframes.
+"""
+
 from datetime import datetime
 
+import epiweeks
 import polars as pl
 
 
-# NOTE: this needs to change to ACTUAL epiweeks
-def calculate_epidate(date):
-    iso_calendar = date.isocalendar()
-    return iso_calendar[1], iso_calendar[0]
+def calculate_epidate(date: str):
```

Comment:
I think the name could be improved. Maybe `calculate_epi_week_and_year` since that's what the function does.

Reply:
Sounds good. My thinking "epidate" captures both but yes is less clear than `calculate_epi_week_and_year`.